rules_bot.py

```diff
@@ -122,23 +122,42 @@ def off_on_topic(bot, update, groups):
         reply = update.message.reply_to_message
         moved_notification = 'I moved this discussion to the ' \
                              '[off-topic Group]({}).'
-        if reply and reply.text:
-            issued_reply = get_reply_id(update)
 
+        if reply and (reply.text or reply.caption):
+            issued_reply = get_reply_id(update)
             if reply.from_user.username:
                 name = '@' + reply.from_user.username
             else:
                 name = reply.from_user.first_name
 
-            replied_message_text = reply.text
+            if reply.photo:
```

Comment:
So what happens if it's a video with a caption?

Reply:
I switched it up a bit, since photos/documents/videos without any caption weren't forwarded. (because of if reply or (reply.text or reply.caption) -> no caption, no forward. 

Checking for reply.text or reply.document, etc. instead. 

I would be happy if you review it again.